src/AtmosphereModels/atmosphere_model.jl

```diff
@@ -100,21 +100,20 @@ function AtmosphereModel(grid;
                          coriolis = nothing,
                          boundary_conditions = NamedTuple(),
                          forcing = NamedTuple(),
-                         advection = nothing,
-                         momentum_advection = nothing,
-                         scalar_advection = nothing,
+                         advection = DefaultValue(),
+                         momentum_advection = DefaultValue(),
+                         scalar_advection = DefaultValue(),
```

Comment:
add explanation for how to use these kwargs or which take precedence over which in the docstring?

Reply:
ok -- but scope creep! this was already present before this PR